ProcessLib/SmallDeformation/CreateSmallDeformationProcess.h

```diff
@@ -70,7 +71,12 @@ createSmallDeformationProcess(
         constitutive_relation_config.peekConfigParameter<std::string>("type");
 
     std::unique_ptr<Solids::MechanicsBase<DisplacementDim>> material = nullptr;
-    if (type == "LinearElasticIsotropic")
+    if (type == "ViscoElasticBurgers")
```

Comment:
why this comes before elastic? I think it's better if we have some rule about the order, e.g. alphabetic order.